Makefile

```diff
@@ -350,7 +350,7 @@ TEST_SRC=./controllers/... ./tests/integration/... ./pkg/...
 .PHONY: envtest
 envtest: $(ENVTEST) ## Download envtest-setup locally if necessary.
 $(ENVTEST): $(LOCALBIN)
-	test -s $(LOCALBIN)/setup-envtest || GOBIN=$(LOCALBIN) go install sigs.k8s.io/controller-runtime/tools/setup-envtest@latest
+	test -s $(LOCALBIN)/setup-envtest || GOBIN=$(LOCALBIN) go install sigs.k8s.io/controller-runtime/tools/setup-envtest@v0.0.0-20240320141353-395cfc7486e6
```

Comment:
May be put the version in a variable somewhere on top?

Reply:
About this... I was thinking, not as part of this PR, but how about putting such variables next to their respective install targets? For me it's somewhat easier to navigate this way.

Reply:
> About this... I was thinking, not as part of this PR, but how about putting such variables next to their respective install targets? For me it's somewhat easier to navigate this way.

My way of thinking is opposite. To change the variable then you should find the place where the install target is. Instead of having them in one visible place on top.